tests/test_helper.tcl

```diff
@@ -588,15 +588,15 @@ proc print_help_screen {} {
         "--single <unit>    Just execute the specified unit (see next option). This option can be repeated."
         "--verbose          Increases verbosity."
         "--list-tests       List all the available test units."
-        "--only <test>      Just execute tests that match <test> regexp. This option can be repeated."
+        "--only <test>      Just execute the specified test by test name or tests that match <test> regexp. This option can be repeated. Regexp must start with '/'."
```

Comment:
```suggestion
        "--only <test>      Just execute the specified test by test name, or tests that match <test> regex (if <test> starts with '/'). This option can be repeated."
```
if this better? if so, apply on the other two.

Reply:
fixed here https://github.com/redis/redis/pull/10741/commits/d3b0364d921384fa1ff66912dc6ddcfb337ea58b